src/test/rpc/Tx_test.cpp

```diff
@@ -1,7 +1,7 @@
 //------------------------------------------------------------------------------
 /*
     This file is part of rippled: https://github.com/ripple/rippled
-    Copyright (c) 2012-2017 Ripple Labs Inc.
```

Comment:
I'm not sure what actual purpose these copyright comments serve legally anyways, but I'm quite certain that they have nothing to do with gRPC.

Reply:
I don't know what they serve either, but one of the other review comments pointed out that the copyright comment had a very old date, so I updated it.